guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -37,66 +37,117 @@ The exported archive file contains the following data:
+
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete version --content-view=view --version=1.0 --organization=export-21527 --chunk-size-mb=10
+[.....................................................................................................................................................................................................................................] [100%]
+Generated /var/lib/pulp/exports/export-21527/view/1.0/2021-02-25T21-15-22-00-00/metadata.json
+
+# ls -lh  /var/lib/pulp/exports/export-21527/view/1.0/2021-02-25T21-15-22-00-00/
+total 49M
+-rw-r--r--. 1 pulp pulp  10M Feb 25 21:15 export-fb548099-7931-4f16-a7b9-7cb47b885037-20210225_2115.tar.gz.0000
+-rw-r--r--. 1 pulp pulp  10M Feb 25 21:15 export-fb548099-7931-4f16-a7b9-7cb47b885037-20210225_2115.tar.gz.0001
+-rw-r--r--. 1 pulp pulp  10M Feb 25 21:15 export-fb548099-7931-4f16-a7b9-7cb47b885037-20210225_2115.tar.gz.0002
+-rw-r--r--. 1 pulp pulp  10M Feb 25 21:15 export-fb548099-7931-4f16-a7b9-7cb47b885037-20210225_2115.tar.gz.0003
+-rw-r--r--. 1 pulp pulp 8.4M Feb 25 21:15 export-fb548099-7931-4f16-a7b9-7cb47b885037-20210225_2115.tar.gz.0004
+-rw-r--r--. 1 pulp pulp  909 Feb 25 21:15 export-fb548099-7931-4f16-a7b9-7cb47b885037-20210225_2115-toc.json
+-rw-r--r--. 1 root root  360 Feb 25 21:15 metadata.json
+----
+
+
+.Incremental Export
+
+Exporting complete versions can be a very expensive operation in terms of space and resources. Content view versions that have multiple RHEL trees may occupy several gigabytes of the space on {ProjectServer}.
+
+{ProjectServer} offers *Incremental Export* to help with this scenario. *Incremental Export* exports only things that changed from the previously exported version. These would be typically smaller than the full exports. In the example below we will incrementally export version `2.0`. Since we have already exported version `1.0` in earlier calls this would only include things that changed between version `1.0` and `2.0`
+
+----
+# hammer content-export incremental version --content-view=view --version=2.0 --organization=export-21527
+[............................................................................................................................................................................................................] [100%]
+Generated /var/lib/pulp/exports/export-21527/view/2.0/2021-02-25T21-45-34-00-00/metadata.json
+
+# ls -lh /var/lib/pulp/exports/export-21527/view/2.0/2021-02-25T21-45-34-00-00/
+total 20M
+-rw-r--r--. 1 pulp pulp 20M Feb 25 21:45 export-ab5b4c58-51eb-49e4-8484-9ff2ea50c122-20210225_2145.tar.gz
+-rw-r--r--. 1 pulp pulp 333 Feb 25 21:45 export-ab5b4c58-51eb-49e4-8484-9ff2ea50c122-20210225_2145-toc.json
+-rw-r--r--. 1 root root 482 Feb 25 21:45 metadata.json
+----
+
+=== Examining the exports
+
+You can query on the exports that you previously have created via the `hammer content-export list` command.
+
+----
+hammer content-export list --organization=export-21527
```

Comment:
This isn't a procedure so it shouldn't have a heading, please add this command to the end of the exporting procedure.